.travis.yml

```
@@ -3,12 +3,13 @@ language: go
 go:
   - 1.10.x
   - 1.11.x
+  - 1.12.x
 
 services:
   - rabbitmq
 
 env:
- - AMQP_URL=amqp://guest:guest@127.0.0.1:5672/
+  - GO111MODULE=on AMQP_URL=amqp://guest:guest@127.0.0.1:5672/
```

I'm not an expert in TravisCI, but I believe this is a list of `KEY=VALUE` pairs.

```suggestion
  - GO111MODULE=on
  - AMQP_URL=amqp://guest:guest@127.0.0.1:5672/
```

---

If you split it out like you've suggested you end up w/ a matrix of builds:

- GO111MODULE=on Go 1.10.x 
- AMQP_URL=amqp://guest:guest@127.0.0.1:5672/ Go 1.10.x 
- GO111MODULE=on Go 1.11.x 
- AMQP_URL=amqp://guest:guest@127.0.0.1:5672/ Go 1.11.x 
...

For a set of environment variables which should apply to a build for each Go version as-is is correct: https://docs.travis-ci.com/user/environment-variables/#defining-multiple-variables-per-item

---

Makes sense.